CtbGui.py

```diff
@@ -1172,7 +1172,12 @@ def update_field(self):
             # For loop stop address
             lineEditLoopStop = getattr(self, f"lineEditLoop{i}Stop")
             lineEditLoopStop.setText(hex((self.det.patloop[i])[1]))
-
+        
+        #OUTPUT SETTINGS
+        self.lineEditFileName.setText(str(self.det.fname))
```

Comment:
Do you need to have the "str" casting for file name and file path? It should already be a string..

Reply:
I have modified the code for fname, not for fpath. fpath returns a PosixPath and needs a cast.